.dev/tests/cypress/helpers.js

```diff
@@ -140,7 +140,9 @@ export function addBlockToPost( blockName, clearEditor = false ) {
 	}
 
 	if ( clearEditor ) {
-		clearBlocks();
+		cy.wrap( null ).then( () =>
```

Comment:
😨

Reply:
I'm going to revert this. Looks like what you tried didn't work either. I'm going to try simply creating a new page.

Reply:
Reverting sounds good. This is strange. I am confused as to why this condition is able to occur. It almost seems like it happens just when clearing the blocks.